tests/modules/nf-core/quilt/quilt/test.yml

```diff
@@ -0,0 +1,133 @@
+- name: quilt quilt test_quilt
+  command: nextflow run ./tests/modules/nf-core/quilt/quilt -entry test_quilt -c ./tests/config/nextflow.config -c ./tests/modules/nf-core/quilt/quilt/nextflow.config
+  tags:
+    - quilt
+    - quilt/quilt
+  files:
+    - path: output/quilt/quilt.chr20.2000001.2100000.vcf.gz
+      md5sum: 81e23143117401f1f37a7c125ce37751
+    - path: output/quilt/quilt.chr20.2000001.2100000.vcf.gz.tbi
+      md5sum: 4607cdcb20599cbebd1ccf76d4dc56ae
+    - path: output/quilt/versions.yml
+
+- name: quilt quilt test_quilt_no_optional_files
+  command: nextflow run ./tests/modules/nf-core/quilt/quilt -entry test_quilt_no_optional_files -c ./tests/config/nextflow.config -c ./tests/modules/nf-core/quilt/quilt/nextflow.config
+  tags:
+    - quilt
+    - quilt/quilt
+  files:
+    - path: output/quilt/quilt.chr20.2000001.2100000.vcf.gz
+      md5sum: 65a415af7fa763422e628c2a418ab189
+    - path: output/quilt/quilt.chr20.2000001.2100000.vcf.gz.tbi
+      md5sum: 20d9e8cda03fc84482f3aa53a0c94fb6
+    - path: output/quilt/versions.yml
+
+- name: quilt quilt test_quilt_optional_outputs
+  command: nextflow run ./tests/modules/nf-core/quilt/quilt -entry test_quilt_optional_outputs -c ./tests/config/nextflow.config -c ./tests/modules/nf-core/quilt/quilt/nextflow.config
+  tags:
+    - quilt
+    - quilt/quilt
+  files:
+    - path: output/quilt/RData/QUILT_prepared_reference.chr20.2000001.2100000.RData
+      md5sum: 74e273dca98556c07e9134107456c4a8
+    - path: output/quilt/plots/haps.NA12878.chr20.2000001.2100000_igs.1.0.truth.png
```

Comment:
Does the md5sum change for all the png ?

Reply:
unfortunately yes